chpl/chpl-resources/src/main/resources/jobs.xml

```diff
@@ -573,6 +573,42 @@
                 </entry>
             </job-data-map>
         </job>
+
+        <!-- Create the Listing Validation Report Data -->
+        <job>
+            <name>listingValidationReportCreator</name>
+            <group>systemJobs</group>
+            <description>Create the listing validation report data</description>
+            <job-class>gov.healthit.chpl.scheduler.job.listingvalidation.ListingValidationCreatorJob</job-class>
+            <durability>true</durability>
+            <recover>false</recover>
+            <job-data-map>
+                <entry>
+                    <key>frequency</key>
+                    <value>DAILY</value>
+                </entry>
+            </job-data-map>
+        </job>
+        
+        <!-- Generate email report for Listing Validation  -->
+        <job>
+            <name>Listing Validation Email Report</name>
+            <group>chplJobs</group>
+            <description>Sends a report providing information about Listing Validation errors</description>
+            <job-class>gov.healthit.chpl.scheduler.job.listingvalidation.ListingValidatorEmailJob</job-class>
+            <durability>true</durability>
+            <recover>false</recover>
+            <job-data-map>
+                <entry>
+                    <key>authorities</key>
+                    <value>ROLE_ADMIN;ROLE_ONC</value>
```

Comment:
This is going to be annoying - how do we reconcile the ticket I have in PR that adds ONC_STAFF to all of these jobs to this PR? Not knowing which will go first.

Reply:
Maybe we can just put the `ONC_STAFF` line in this PR now? I don't think it'll break anything if it's there, it just might not matter if that ROLE_ONC ticket isn't in place yet

Reply:
I would prefer that!